restServices/pom.xml

```diff
@@ -59,7 +59,7 @@
 		<dependency>
     		<groupId>mysql</groupId>
     		<artifactId>mysql-connector-java</artifactId>
-    		<version>8.0.16</version>
+    		<version>8.0.12</version>

```

Comment:
Why are we downgrading to the lower version of mysql-connector

Reply:
is this code tested ?

Reply:
you have downgraded the version!!